:groovyDoc task is broken

### Describe the bug

Since commit 871a2115670e04876c739dc5ff6b49048fe70ae5 which updated to Gradle 7.6, the `:groovyDoc` task is failing with

```plain
Some problems were found with the configuration of task ':groovydoc' (type 'Groovydoc').
  - Type 'org.gradle.api.tasks.javadoc.Groovydoc' property 'access' doesn't have a configured value.

    Reason: This property isn't marked as optional and no value has been configured.

    Possible solutions:
      1. Assign a value to 'access'.
      2. Mark property 'access' as optional.

    Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'org.gradle.api.tasks.javadoc.Groovydoc' property 'includeAuthor' doesn't have a configured value.

    Reason: This property isn't marked as optional and no value has been configured.

    Possible solutions:
      1. Assign a value to 'includeAuthor'.
      2. Mark property 'includeAuthor' as optional.

    Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'org.gradle.api.tasks.javadoc.Groovydoc' property 'includeMainForScripts' doesn't have a configured value.

    Reason: This property isn't marked as optional and no value has been configured.

    Possible solutions:
      1. Assign a value to 'includeMainForScripts'.
      2. Mark property 'includeMainForScripts' as optional.

    Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#value_not_set for more details about this problem.
  - Type 'org.gradle.api.tasks.javadoc.Groovydoc' property 'processScripts' doesn't have a configured value.

    Reason: This property isn't marked as optional and no value has been configured.

    Possible solutions:
      1. Assign a value to 'processScripts'.
      2. Mark property 'processScripts' as optional.

    Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#value_not_set for more details about this problem.
```

### To Reproduce

./gradlew :groovyDoc

### Expected behavior

> BUILD SUCCESSFUL

### Actual behavior

> BUILD FAILED

--- a/build.gradle
+++ b/build.gradle
@@ -366,6 +366,11 @@ tasks.register("groovydoc", Groovydoc) {
   source subprojects.collect { project -> project.tasks.named("groovydoc").map { it.source } }
   classpath = files(subprojects.collect { project -> project.tasks.named("groovydoc").map { it.classpath } })
   groovyClasspath = files(projects.spockCore.dependencyProject.tasks.named("groovydoc").map { it.groovyClasspath })
+
+  access = GroovydocAccess.PROTECTED
+  processScripts = false
+  includeMainForScripts = false
+  includeAuthor = false
 }
 
 configureGroovydoc(tasks.named("groovydoc"))

